robots_101/team_supervisor.md

```diff
@@ -17,7 +21,18 @@ You’ll be our point of contact with the team. If you have any questions during
 We aim to host our Kickstart event and Tech Days in multiple locations to make it more convenient for you to travel. However, you will still need to arrange to travel to these places.
 This is especially important to book for the competition, as you will likely need to arrange to stay overnight near the venue.
 
-Our kit includes a battery and a few boards to get your robot started. However, your team will need additional components and materials from which to build their robot. For the chassis, your team has several options. Teams often use cardboard, MDF, Aluminium, and/or Acrylic. In terms of electronic components, teams often make use of:
+## The team
```

Comment:
Not super happy with this section heading name; would appreciate some suggestions.